wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/CellDescription/MLSMigrationCellDescription.swift

```diff
@@ -22,12 +22,18 @@ import WireDataModel
 final class MLSMigrationCellDescription: ConversationMessageCellDescription {
 
     typealias View = ConversationSystemMessageCell
+    typealias SystemMessageMLSMigrationLocalizable = L10n.Localizable.Content.System.MlsMigration
 
     private static let linkAttributes: [NSAttributedString.Key: Any] = [
         .font: UIFont.mediumSemiboldFont,
         .link: URL.wr_mlsLearnMore
     ]
 
+    private static let linkAttributesForDownloadingWire: [NSAttributedString.Key: Any] = [
+        .font: UIFont.mediumSemiboldFont,
+        .link: URL.wr_wireAppOnItunes
+    ]
+
```

Comment:
Is `linkAttributesForDownloadingWire` still needed in this place?

Reply:
Nope, I will delete it from here.